knowledge-base/grid-custom-filter-menu.md

```diff
@@ -0,0 +1,201 @@
+---
+title: How to Refresh Filter Menu Context After Programmatic Changes
```

Comment:
Aren't we refreshing the Filter Menu UI? The context is always up-to-date, but the users don't see it.